pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -2236,6 +2236,9 @@
         "4.5.1"
       ],
       "BaselineVersion": "4.4.0",
+      "InboxOn": {
+        "uap10.0.16300": "4.0.3.0"
```

Comment:
Yes that's the actual TFM. It's the three-part Windows SDK number.

Reply:
I don't think this will be the actual TFM we should use. RS4 shipped with a version higher than 10.0.16300 and that one is still only supporting uap6.1. If we need any changes in Windows we should change the TFM number to match whatever Windows Release will contain them.